ONSdigital/dp-frontend-filter-flex-dataset
# Feature/handle dimension name route

### What

WIP to handle successful filter/flex job and redirect to `flex/{filterId}/dimensions/{name}`

**Note**
The POST endpoint used to create the `filterId` is not ready/being consumed yet

### How to review

- Sense check
- Test pass
- Run this branch of the service and the `dp-design-system`
- Go to [http://localhost:20100/flex/{dimensionId}/dimensions/{name}](http://localhost:20100/flex/{dimensionId}/dimensions/{name}) replace `dimensionId` and `name` with values of your choice
- The page should load (200) and the `title` should populate with the `{name}` value

### Who can review

Frontend dev
